loky/backend/queues.py

```diff
@@ -273,88 +165,42 @@ def _feed(buffer, notempty, send_bytes, writelock, close, reducers,
             # We ignore errors which happen after the process has
             # started to cleanup.
             try:
-                if is_exiting():
-                    info('error in queue thread: %s', e)
+                if util.is_exiting():
+                    util.info('error in queue thread: %s', e)
                 else:
                     import traceback
                     traceback.print_exc()
             except Exception:
                 pass
 
+    if sys.version_info[:2] < (3, 4):
+        # Compat for python2.7/3.3 that use _send instead of _send_bytes
+        def _after_fork(self):
+            super(Queue, self)._after_fork()
+            self._send_bytes = self._writer.send_bytes
 
-_sentinel = object()
 
-#
-# A queue type which also supports join() and task_done() methods
-#
-# Note that if you do not call task_done() for each finished task then
-# eventually the counter's semaphore may overflow causing Bad Things
-# to happen.
-#
-
-
-class JoinableQueue(Queue):
```

Comment:
I removed our `JoinableQueue` implementation as we do not use it anywhere and it adds extra maitainance burden.